xcat-inventory/xcclient/inventory/backend.py

```diff
@@ -0,0 +1,695 @@
+#!/usr/bin/python
+from __future__ import print_function
+import 	configparser
+import os
+import sh
+import shutil
+import re
+import sys
+from exceptions import * 
+import manager 
+import utils
+import pickle
+import hashlib
+
+class Invbackend:
```

Comment:
Use the new type of inheritance `class Invbackend(object)`

Reply:
fixed